crunchy-xml-decoder/hls.py

```diff
@@ -71,7 +71,7 @@ def copy_with_decrypt(input, output, key):
 def fetch_streams(output, video):
     output = open(output, 'wb')
     for n, seg in enumerate(video.segments):
-        sys.stdout.write('\x1b[2K\r%d/%d' % (n + 1, len(video.segments)))
+        sys.stdout.write('\r%d/%d' % (n + 1, len(video.segments)))
```

Comment:
This part I actually don't like. Given that I often add some additional data, the escape sequences clean the left overs.